test/box_test.dart

```diff
@@ -93,6 +93,18 @@ void main() {
       expect(fetchedItems[2].text, equals("Two"));
     });
 
+    test(".count(), .isEmpty(), .removeAll() works", () {
+      final List<TestEntity> items = ["One", "Two"].map((s) => TestEntity.construct(s)).toList();
+      final List<int> ids = box.putMany(items);
+      
+      int count = box.count();
+      expect(count, greaterThanOrEqualTo(0));
+      bool isEmpty = box.isEmpty();
+      expect(isEmpty, equals(false));
+      int removed = box.removeAll();
+      expect(removed, greaterThan(0));
+    });
+
```

Comment:
Could you split this up to separate tests (per function)? Also, it should be a little more thorough, these are fairly important methods :)

e.g. sth like the following for `count()`:
```dart
    test("count", () {
      expect(box.count(), equals(0));
      final List<TestEntity> items = ["One", "Two"].map((s) => TestEntity.construct(s)).toList();
      box.putMany(items);
      
      expect(box.count(), equals(2));
    });
```